common/build-style/haskell-stack.sh

```diff
@@ -4,6 +4,9 @@
 # make_build_args="stack-build-flags"
 # stackage="lts-X.Y"  # or include a stack.yaml in $FILESDIR
 #
+
+export STACK_ROOT="/host/stack/${XBPS_TARGET_MACHINE}"
```

Comment:
We don't put environment variables here. Use `common/environment/build-style/haskell-stack.sh`